src/components/Dashboard/Capacity/CapacityItem.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { DonutChart } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import { InlineLoading } from '../../Loading';
+import { prefixedId } from '../../../utils';
+import { USED, AVAILABLE, LOADING, CAPACITY } from './strings';
+
+const pfGetUtilizationDonutTooltipContents = d =>
+  `<span class="donut-tooltip-pf" style="white-space: nowrap;">${Math.round(d[0].ratio * 1000) / 10} ${
+    d[0].name
+  }</span>`;
+
+export const CapacityItem = ({ id, title, used, total, unit, formatValue, LoadingComponent }) => {
+  let description = <LoadingComponent />;
+  let columns = [[USED, 0], [AVAILABLE, 100]];
+  let tooltip;
+  let primaryTitle = LOADING;
+  let secondaryTitle = CAPACITY;
+  if (used != null && total != null) {
+    const totalFormatted = formatValue(total, unit);
+    const usedFormatted = formatValue(used, totalFormatted.unit);
+
+    unit = totalFormatted.unit; // eslint-disable-line prefer-destructuring
+    total = totalFormatted.value;
+    used = usedFormatted.value;
+
+    const available = total - used;
+    columns = [[USED, used], [AVAILABLE, available]];
+
+    const percentageUsed = total > 0 ? Math.round((100 * used) / total) : 0;
+    const availableNice = Number.isInteger(available) ? available : available.toFixed(1);
+    const totalNice = Number.isInteger(total) ? total : total.toFixed(1);
+
+    description = `${availableNice} ${unit} available out of ${totalNice} ${unit}`;
+    tooltip = { contents: pfGetUtilizationDonutTooltipContents };
+    primaryTitle = `${percentageUsed.toString()}%`;
+    secondaryTitle = USED;
+  }
+
+  return (
+    <div className="kubevirt-capacity__item">
+      <h2 className="kubevirt-capacity__item-title">{title}</h2>
+      <h6>{description}</h6>
+      <div>
+        <DonutChart
+          id={prefixedId('donut-chart', id)}
```

Comment:
I just didn't want to force the `donut-chart` part on the consumer. Nevertheless it probably doesn't matter that much.